mmv1/third_party/terraform/website/docs/r/compute_region_instance_group_manager.html.markdown

```diff
@@ -42,7 +42,7 @@ resource "google_compute_region_instance_group_manager" "appserver" {
   distribution_policy_zones  = ["us-central1-a", "us-central1-f"]
 
   version {
-    instance_template = google_compute_instance_template.appserver.id
+    instance_template = google_compute_instance_template.appserver.self_link_unique
```

Comment:
Also consider document that self_link_unique is preferred for instance_template.

Reply:
Thanks, added a remark.